sql-scripts/create-authentication.sql

```diff
@@ -0,0 +1,37 @@
+USE TheITCrowd_MiaG;
+
+CREATE TABLE Users (
+    userId INT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
+    email VARCHAR(50) UNIQUE NOT NULL,
+    passwordHash TEXT NOT NULL,
+    firstName VARCHAR(50) NOT NULL,
+    lastName VARCHAR(50) NOT NULL,
+    accessRight ENUM('Dummy', 'Blocked', 'View', 'Admin') DEFAULT 'View' NOT NULL
```

Comment:
This makes it very difficult to add/remove any of these at a later stage.

It is also not required for this ticket

Reply:
I don't think we will ever need any others at a later stage. Would you be happy to keep these the way they are. It was necessary to store as part of JWT requirement... (Not technically part of ticket) - trying to set a standard for further implementation.  :)